packages/host/app/components/operator-mode/code-submode.gts

```diff
@@ -468,6 +469,13 @@ export default class CodeSubmode extends Component<Signature> {
     }
   }
 
+  get showBoxelSpecPreview() {
+    return (
+      !this.moduleContentsResource.isLoading &&
+      this.selectedDeclaration?.exportName
```

Comment:
Should boxel spec pane be present for any module, or just for cards? What does this logic say?

Reply:
In this logic, it models the boxel spec as present in any selected code with an exportName (including defaults)
- exported card 
- exported field 

However,  at the top-level, exported variable, function, class which are not card or fields, are ignored entirely by the RHS

<img width="1894" alt="Screenshot 2024-12-31 at 13 33 50" src="https://github.com/user-attachments/assets/7fecd7f4-66e9-48af-9342-174a3c9beb26" />